docs/icicle/integrations.md

```diff
@@ -0,0 +1,22 @@
+# Integrations
+
+ICICLE has been used by companies like [Celer Network](https://github.com/celer-network), [Gnark](https://github.com/Consensys/gnark), [EZKL](https://blog.ezkl.xyz/post/acceleration/) and others to accelerate their ZK proving pipeline.
+
+Many of these integrations have been a collaboration between Ingonyama and the integrating company. We have learned a lot about designing GPU based ZK provers.  
+
+If your interested in understanding these integrations better and learning how you can use ICICLE to accelerating your existing ZK proving pipeline this is the place for you.
+
+Lets illustrate an ICICLE integration, so you can understand the core API and design overview of ICICLE.
+
+![ICICLE architecture](../../static/img/architecture-highlight.png)
```

Comment:
The image shows that only Rust and Golang APIs are highlevel APIs, but the c++ wrappers/API is also a highlevel api in some sense. For an application that is in c/c++, there is no need to use Rust/Golang APIs, it can use the c++ directly.